apps/web/src/features/terminal/components/TerminalAgentInputOverlay.css

```
@@ -144,6 +139,24 @@
   }
 }
 
+.terminal-agent-selector-attention {
+  animation: terminalAgentSelectorShake 400ms ease-in-out;
+  border-radius: 8px;
+  box-shadow:
+    0 0 0 2px color-mix(in srgb, #f59e0b 60%, transparent),
+    0 0 12px color-mix(in srgb, #f59e0b 25%, transparent);
+}
+
+@keyframes terminalAgentSelectorShake {
```

Stylelint (17.14.0)

[error] Expected keyframe name "terminalAgentSelectorShake" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)